src/domains/sign/io_sign_user_func.tf

```diff
@@ -0,0 +1,61 @@
+locals {
```

Comment:
does the function deploy mechanism rely on swapping slots? (I'm asking since I can't see the slot configuration)